### ISSUE TITLE
V9 errors and issues

### ISSUE THREAD

**Issue by rockwallguy** [trioderegion/dnd5e-helpers#229]
I know that 5e helpers hasn't been updated for v9, but I've been testing things and thought I'd let you know of some issues that I noticed.  The first was mentioned before and closed, but I'm getting this error.

![image](https://user-images.githubusercontent.com/79724010/146985160-d645e50c-48cb-4a80-ac54-692ac7dada8e.png)

The second issue (unsure if related) is that if any status effect is on the token, the visual indicator of that status effect vanishes and I lose the ability to interact with that token.  This happens whether the status effect is from DFreds Convienent Effects or from just the base foundry tokens.  It is easily repeatable.  Simply turn off 5e helpers, apply a status effect, then reactivate 5e helpers.  It's happened to me every time.

I've also had an issue where I lose the linking of a token to its respective actor, and that happened while the only "not ready for v9" module I had active was 5e helpers.  This happened to several of the tokens on my active scene while I was testing the above.  I can't reliably duplicate this bug, and I may be wrong about the source, but I thought it was worth mentioning.

**Comment by trioderegion** [comment]
That last one shouldn't be due to helpers. Was that token spawned with Warp Gate by any chance?

**Comment by rockwallguy** [comment]
It wasn't.  It was just a default drag and drop.  The more I think about it, those tokens were on the scene when I updated to v9.  And the only tokens that didn't de-link were the ones that had status icons on them while I was testing the bug I described.  They might have just de-linked during the update process or something.  It might not be related to your module.  There were 5 tokens on the scene, all linked to the same actor, and two that had status icons didn't de-link, but the others did.  It may not be relevant to 5e helpers.  I just noticed it during my testing, and thought I'd mention it.
